src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Utilities.cs

```diff
@@ -47,7 +48,10 @@ public static void ExtractTarball(string tarballPath, string outputDir, string t
         }
     }
 
-    public static IEnumerable<string> GetTarballContentNames(string tarballPath)
+    public static IEnumerable<string> GetTarballContentNames(string tarballPath) =>
+        GetTarballContent(tarballPath).Select(entry => entry.Name);
+
+    public static IEnumerable<TarEntry> GetTarballContent(string tarballPath)
```

Comment:
Only SdkContentTests utilizes GetTarballContentNames. IMO, I think having the GetTarballContent base functionality is useful in case future tests need to use it.

Reply:
But it's an used public surface area.  Yes it's minor but if this pattern were repeated everywhere it makes the codebase rather messy.  If it is needed in the future the refactoring is trivial

Reply:
Reverted to the original code [here](https://github.com/dotnet/installer/pull/17233/commits/f891d665737a4b2f69336eb187a21557014fd0f9#diff-3e52a7713dbdb53496cf7dc21844b2eb5e5934d94c9cd8f54d30106af2f0d21fR60).